crates/nargo_cli/tests/test_data_ssa_refactor/to_bytes_integration/src/main.nr

```diff
@@ -0,0 +1,27 @@
+use dep::std;
+
+fn main(x : Field, _y: Field)  {
+    // The result of this byte array will be big-endian
+    let y: Field = 2040124;
+    let be_byte_array = y.to_be_bytes(31);
+    // The result of this byte array will be little-endian
+    let le_byte_array = x.to_le_bytes(31);
```

Comment:
Looks like we can remove `_y` as well as the comments here. I think the variable names and function calls are enough to clue into which is big endian and which is little endian.